src/App.tsx

```diff
@@ -32,7 +32,27 @@ const link = from([
 
 const client = new ApolloClient({
   link,
-  cache: new InMemoryCache(),
+  cache: new InMemoryCache({
+    typePolicies: {
+      Query: {
+        fields: {
+          pokemon_v2_pokemon: {
+            keyArgs: false,
+            //TODO: remove any after merge generates types
```

Comment:
👀

Reply:
It's Apollo type, I have no idea how to adds type here